docs/ecctl_platform_proxy_settings_update.md

```diff
@@ -0,0 +1,45 @@
+## ecctl platform proxy settings update
+
+Updates proxies settings (Available for ECE only)
```

Comment:
```suggestion
Updates proxy settings (Available for ECE only)
```

Reply:
@alaudazzi this operation changes the settings for all proxies, not a single one. Do we still want to use the singular form?

Reply:
What about 
```suggestion
Updates settings for all proxies (Available for ECE only)
```